lib/widgets/my_application.cc

```diff
@@ -0,0 +1,107 @@
+#include "my_application.h"
```

Comment:
Pretty sure this file is here by mistake. There should only be dart code in the lib folder.

Reply:
Oops, its been removed now